unstructured/utils.py

```diff
@@ -50,3 +51,26 @@ def dependency_exists(dependency):
     except ImportError:
         return False
     return True
+
+
+# Copied from unstructured/ingest/connector/biomed.py
+def validate_date_args(date):
+    date_formats = ["%Y-%m-%d", "%Y-%m-%dT%H:%M:%S", "%Y-%m-%dT%H:%M:%S%z"]
+
+    valid = False
+    if date:
```

Comment:
i'd do `if not date` and just raise a ValueError here, and avoid the extra nesting.